### PR TITLE
feat: support customized nodePort in ui service

### PR DESCRIPTION
**Pull request by jhandguy**
Changes proposed in this PR:
- add optional nodePort for http or/and https ports in ui service

Checklist:
- [x] Bats tests added

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -984,6 +984,19 @@ ui:
     # @type: string
     type: null
 
+    # Optionally hardcode the nodePort of the ui service if using a NodePort service.
```
Comment (thisisnotashwin) [inline values.yaml:987]:
```suggestion
    # Optionally set the nodePort value of the ui service if using a NodePort service.
```

**Review by thisisnotashwin — APPROVED** [review]
This is excellent!! Thanks so much for the contribution @jhandguy! Have suggested a minor update to the comment but it looks great otherwise!!

**Review by ishustava — APPROVED** [review]
Looks great!

**Comment by thisisnotashwin** [comment]
Thanks again for the contribution @jhandguy !! Have a great weekend!

**Pull request merged**